engineio/async/eventlet.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
```

Comment:
I think there is some odd interaction with `importlib` that is causing these problems, because 2.7 supports absolute imports just fine. I'll try a few things to see why this happens, I think this should work without using the future import.